addons/mail/static/tests/messaging_menu/messaging_menu_tests.js

```diff
@@ -146,6 +147,10 @@ QUnit.test("rendering with PWA installation request", async (assert) => {
     });
 
     const { env } = await start();
+    // This event must be triggered to initialize the installPrompt service properly
+    // as if it was run by a browser supporting PWA (never triggered in a test otherwise).
+    window.dispatchEvent(new CustomEvent("beforeinstallprompt"));
+
```

Comment:
Why not dispatch on `browser` instead of `window`?
`browser` is safer to target the tests, whereas `window` could target the test environment itself, resulting to undesirable side-effects